data/methane_onestep.yaml

```diff
@@ -0,0 +1,104 @@
+generator: ctml2yaml
+cantera-version: 2.5.0a4
+
+phases:
+- name: methane_onestep
+  elements: [O, H, C]
+  species: [O2, H2O, CH4, CO2]

```

Comment:
You could make this input file significantly shorter by using
```yaml
  species:
  - gri30.yaml/species: [O2, H2O, CH4, CO2]
```
(same for `methane_twostep.yaml`) unless there is a critical difference. If these are just for internal testing, these files should probably go to `test/data`.

Reply:
They are for internal testing. I will move them. Thank you!

Reply:
>  Would that eliminate the need for the species descriptions?

Correct.